pandas/core/indexes/datetimelike.py

```diff
@@ -777,6 +777,12 @@ def wrap_arithmetic_op(self, other, result):
     if result is NotImplemented:
         return NotImplemented
 
+    if isinstance(result, tuple):
+        # divmod, rdivmod
+        assert len(result) == 2
```

Comment:
Would this bubble up in any way?

Reply:
Referring just to the `len-2`.  While I appreciate dummy-protection, I would still want to know if that `assert` would surface for end-users, that's all.